edx_proctoring/api.py

```diff
@@ -2943,7 +2943,7 @@ def get_student_view(user_id, course_id, content_id,
     is_proctored_exam = exam['is_proctored'] and not exam['is_practice_exam']
     is_timed_exam = not exam['is_proctored'] and not exam['is_practice_exam']
 
-    exam_backend = get_backend_provider(name=exam['backend'])
+    exam_backend = get_backend_provider(exam=exam)
```

Comment:
If an exam is passed in, as opposed to the provider name, a timed exam will always return "None".